### DISCUSSION TITLE
Difference - remove from radarr / remove data ?

### DISCUSSION THREAD

**Discussion by HanaO00** [seerr-team/seerr#412] [Q&A]
Hello, what is the difference between these two options?

- Remove from Radarr
- Erase data

When I use the second option, the movie is deleted from Jellyseerr but no radar, nor the mkv file, is this normal?

When I use remove from radarr, the movie removes itself from radarr, but only from radarr and not in torrent client, yet it says "including all files"

![Capture d’écran_2023-06-19_09-39-19](https://github.com/Fallenbagel/jellyseerr/assets/17810985/7f907385-2fca-4523-af9e-c276852f8ee4)

Thanks

**Comment by fallenbagel** [comment]
Both buttons are working as intended.
Remove from radarr button removes the movie file from radarr. Clear data is to clear data from jellyseerr. Jellyseerr does not interact with the Torrent client or any download managers

> including all files

Refers to the movie folder if it has any nfos or images and stuff

**Comment by HanaO00** [comment]
Thank you for your help,

Is it possible for the series to add an option to choose the type of series manually: anime/series?

Because for animated series it is chosen automatically but I have no results in my indexer because the episodes are in SXXEXX

And if I change manually in sonarr I get results
